src/plugins/intel_gpu/tests/test_cases/loop_gpu_test.cpp

```diff
@@ -91,38 +110,47 @@ TEST(loop_gpu, basic_no_concat)
 
     // value check
     {
-        mem_lock<float> output_ptr{ output, get_test_stream() };
-        ASSERT_EQ(output_ptr.size(), input_data.size());
+        mem_lock<T> output_ptr{ output, get_test_stream() };
+        EXPECT_EQ(output_ptr.size(), input_data.size());
         for (size_t i = 0, iend = input_data.size(); i < iend; ++i) {
             ASSERT_FLOAT_EQ(output_ptr[i], input_data[i] + eltwise_operand[i] * trip_count);
         }
     }
 
     // allocate new output memory
-    layout loop_l = network.get_output_memory("loop")->get_layout();
+    layout loop_l = network->get_output_memory("loop")->get_layout();
     auto output_mem = engine.allocate_memory(loop_l);
-    network.set_output_memory("loop", output_mem);
+    network->set_output_memory("loop", output_mem);
 
     //one more execute
     set_values(input_mem, input_data);
     set_values(operand_mem, eltwise_operand);
     set_values(trip_count_mem, { trip_count });
     set_values(initial_condition_mem, { initial_condition });
-    outputs = network.execute();
+    outputs = network->execute();
 
     // check everything once again
     ASSERT_EQ(outputs.size(), 1);
     auto output2 = outputs.begin()->second.get_memory();
     {
-        mem_lock<float> output_ptr2{ output2, get_test_stream() };
-        ASSERT_EQ(output_ptr2.size(), input_data.size());
+        mem_lock<T> output_ptr2{ output2, get_test_stream() };
+        EXPECT_EQ(output_ptr2.size(), input_data.size());
```

Comment:
ASSERT_EQ

Reply:
I replaced all EXPECT_EQ with ASSERT_EQ. Thank you.